src/Backend/Core/Engine/Meta.php

```diff
@@ -477,6 +477,11 @@ public function save($update = false)
             $custom = null;
         }
 
+		//serialize data for save
+        if(!empty($this->data['data'])){
+            $this->data['data'] = serialize($this->data['data']);
```

Comment:
@carakas the database inserting/updating is just below it,
so I see no reason why it may not be serialized at this line?

It is a valid bugfix, I tried it over here, because I had the same problem.